scripts/model-writer.js

```diff
@@ -77,6 +80,86 @@ function remove_model_property(ele){
     }
 }
 
+ModelWriter.ucfirst = function(string)
+{
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+ModelWriter.write = function(isPlainText)
+{
+    var formData = ModelWriter.readForm();
+    var language = formData.language;
+    var company = formData.company;
+    var model = formData.model;
+    var namespace = formData.namespace;
+    var args = formData.args;
+    var output = "";
+    var nl = "<br>";
+    var sp = "&nbsp;";
+    var tab = sp + sp + sp + sp;
+    if (isPlainText) {
+        nl = "\n\r";
+        sp = " ";
+        tab = "    ";
+    }
+    switch (language) {
+        case "php":
+            output = "/*" + nl
+                   + " * (c) " + company + nl
+                   + " *" + nl
+                   + " * For the full copyright and license information, please view the LICENSE" + nl
+                   + " * file that was distributed with this source code." + nl
+                   + " */<br>" + nl
+                   + "namespace " + namespace + nl
+                   + nl
+                   + "/**" + nl
+                   + " * @author Marie Kauth <marie.kauth@gmail.com>" + nl
+                   + " */" + nl
+                   + "class " + model + nl
+		           + "{" + nl
+            for (var arg of args) {
+                var name = arg.name;
+                var type = arg.type;
+                output += nl
+                        + tab + "/** @param " + type + " $" + name + "*/" + nl
```

Comment:
One small update here. can you please add a space before */

                        + tab + "/** @param " + type + " $" + name + " */" + nl